services/credential-server/src/modules/signify/signifyApi.ts

```diff
@@ -415,4 +438,42 @@ export class SignifyApi {
     }
     return op;
   }
+
+  async saidifySchema(schema: any, label?: string): Promise<any> {
+    const customizableKeys: { [key: string]: any } = {};
+    const removeCustomizables = (obj: any, path: string[] = []): any => {
```

Comment:
I think I need to understand this customizable part first before I understand what's happening here

Reply:
Mentioned in other comments but I can provide more information if needed :)